packages/babel-preset-react-app/index.js

```diff
@@ -92,7 +92,14 @@ module.exports = function(api, opts) {
       // don't work without it: https://github.com/babel/babel/issues/7215
       require('@babel/plugin-transform-destructuring').default,
       // class { handleClick = () => { } }
-      require('@babel/plugin-proposal-class-properties').default,
+      // Enable loose mode to use assignment instead of defineProperty, as some
+      // libraries add getters and setters to prototypes
```

Comment:
Let's change this to 

```
// Enable loose mode to use assignment instead of defineProperty
// See discussion in https://github.com/facebook/create-react-app/issues/4263
```